docs/cloud/earthly-ci.md

```diff
@@ -0,0 +1,149 @@
+
+<!-- Use this after alpha.
+Fill out the [Earthly CI signup form](https://earthly.dev/signup/earthly-ci/). You will need to wait for an Earthly team member to approve your request.
+-->
+
+### 3. Ensure that you have the latest version of Earthly
+
+You need to make sure that you are using the [latest Earthly 0.7 release candidate](https://github.com/earthly/earthly/releases). Since this is a release candidate, it will not show up in the regular update channels, such as brew. You have to download the binary manually and add it to your `PATH`. Please note that there are some key changes from 0.6 to 0.7 - one of the important ones is that the cloud-based secret storage has a new format. See the [secrets migration guide](./cloud-secrets.md#migrating-from-the-old-0.6-experimental-version-of-earthly-secrets) for more information.
+
+<!-- Use this after alpha.
+Because this feature is under heavy development right now, it is very important that you use the latest version of Earthly available.
+
+**On Linux**, simply repeat the [installation steps](https://earthly.dev/get-earthly) to upgrade.
+
+**On Mac**, you can perform:
+
+```bash
+brew update
+brew upgrade earthly/earthly/earthly
+```
+-->
+
+### 4. Open the Earthly Web UI
+
+You can open the Earthly Web UI by running:
+
+```bash
+earthly web
+```
+
+This will associate your Earthly account with your GitHub login and then take you to the Earthly Web UI.
+
+### 5. Create your first Earthly pipeline
+
+In the Earthly Web UI, under the Organizations drop-down in the top-left of the screen, select the organization that was granted access to Earthly CI. Then follow the instructions on the screen to create a new Earthly CI project, add your first repository, create your pipeline via a new or an existing Earthfile, and then run your first build.
+
+For your very first CI pipeline you can use the following example:
+
+```Earthfile
+VERSION 0.7
+PROJECT my-org/my-project
+
+FROM alpine:3.15
+
+my-pipeline:
+  PIPELINE
+  TRIGGER push main
+  TRIGGER pr main
+  BUILD +my-build
+
+my-build:
+  RUN echo Hello world
+```
+
+This example shows a simple pipeline called `my-pipeline`, which is triggered on either a push to the `main` branch, or a pull request against the `main` branch. The pipeline executes the target `my-build`, which simply prints `Hello world`.
+
+Pipelines and their definitions, including their triggers must be merged into the primary branch (which, unless overridden, is the default branch on GitHub -- usually `main`) in order for the triggers to take effect.
+
+Please note that the `PROJECT` declaration needs to match the name of your organization and the name of your Earthly project that you create in the Earthly Web UI.
+
+### 6. Set up registry access
+
+During your experimentation with Earthly CI, you may encounter DockerHub rate limiting errors. To avoid this, you can log into your DockerHub account by using the command `earthly registry setup`.
```

Comment:
```suggestion
During your experimentation with Earthly CI, you may encounter DockerHub rate limiting errors. To avoid this, you can setup your DockerHub account by using the command `earthly registry setup`.
```